tasks/install-FreeBSD.yml

```diff
@@ -34,8 +34,15 @@
     name: "{{ postfix_service }}"
     enabled: yes
 
-# TODO periodic
-# daily_clean_hoststat_enable="NO"
-# daily_status_mail_rejects_enable="NO"
-# daily_status_include_submit_mailq="NO"
-# daily_submit_queuerun="NO"
+- name: Disable sendmail-specific tasks
+  lineinfile:
+    dest: /etc/periodic.conf
+    regexp: "^{{ item }}="
```

Comment:
- does regexp: insert the line if not found?
- regexp: 'only the last line found will be replaced.'